### ISSUE TITLE
Leading zeros in IP address; cluttered callstring, nonuniform reply by WiFi and UDP Appsets 

### ISSUE THREAD

**Issue by arehmer** [HeimannSensor/ESP32_ApplicationShield#4]
In the package returned by the Appset upon 'Calling HTPA series devices' the following string is returned (example):

HTPA series responsed! I am Arraytype 14 MODTYPE 011
ADC: 16
HTPAxxxxxd v.2.00 Heimann Sensor GmbH; written by D. Pauer 2021-01-13
I am running on 5000.0 kHz
MAC-ID: 00.00.00.00.00.00 IP: 192.168.137.098 DevID: 000000654

1. The unnecessary amount of text makes automatic processing of the string unnecessary complicated. A simple message like

Arraytype: 14; MODTYPE: 11; ADC: 16, Clock: 5000.0; 

would be much better than 'I am running at ...'

2. Although it's not wrong, leading zeros in IP addresses should be avoided since they can be confused with octal representations and are handled differently by different operating systems. Windows for example does not find the device above via the IP 192.168.137.098 but only via the IP 192.168.137.98 

In short: The string should be modified to have a simple and clear structure and leading zeros should be avoided.

Also: The callstrings of the WiFi and UDP Appset differ from each other. This forces the customer to write different code for different appsets.
